### DISCUSSION TITLE
"Biogenic" flows in the flowlist

### DISCUSSION THREAD

**Discussion by bl-young** [USEPA/Federal-LCA-Commons-Elementary-Flow-List#179] [General]
Users regularly inquire about how to track biogenic flows (e.g., "Carbon dioxide, biogenic"). The FEDEFL does not contain any biogenic flows. This discussion is intended to document that decision and receive feedback.

**Comment by ErCollao** [comment]
Thank you for opening this @bl-young , and for all the effort that you guys do here!

I can share our experience with this type of flows. We are big fans of how your flowlist cleanly structures elementary flows. It's much more homogeneous than other elementary flow lists. For that reason, [we](earthster.org) use it as our standard nomenclature, and interpret all data (e.g. datasets like Ecoinvent, or methods like ReCiPe, PEF, etc) using your flowlist.

We've encountered that any information about the "biogenic" nature of greenhouse emissions gets lost in this translation. Ecoinvent has such information, but we can't represent it (to our knowledge) in terms of the flowlist. In some impact assessment methods, we would also have those flows be treated differently for different impact categories (e.g. the most common modelings of IPCC for LCA). We of course can't do that if that information gets lost.

Do I understand correctly that there's a formal decision _not_ to include biogenic flows? Can you share what the rational is for such a decision?

**Reply by fchichorro** [reply]
Hi! I was the author of the PR to include biogenic and biomass carbon dioxide (#178). Thank you @bl-young for reviewing it so quickly and initiating this discussion.

Is the decision not to include biogenic carbon in the flow list a short-term or a long-term decision? Asking because at some point in a shorter time frame we will probably have to add biogenic carbon, but if we can follow FEDEFL as strictly as possible then we preferably will.

I share @ErCollao's thoughts about FEDEFL, I really like its consistency and coherency. Thank you for all the nice work!

**Comment by ErCollao** [comment]
We're contemplating different ways of handling this context, but some of them may be relevant for FEDEFL itself. The ideas so far have been:
* Add "biogenic" and "land use" versions of Greenhouse gases, like almost all elementary flow lists have done. I'm assuming this is something you have rejected in the past. I'm assuming (please correct me if I'm wrong) that this is because the _flowable_ itself is not different, chemistry-wise, so conceptually it makes the flowable list messier. If that's the case, we can look for more complex solutions (see below).
* Add a new dimension "tag(s)", basically specifying traits of the elementary flow in question. Default tags (for everything) could be "unspecified" (or just empty), but there could be elementary flows defined as e.g. `flowable: "Carbon dioxide", context: "emission/air", unit: "kg", tag: "biogenic"`. The idea behind naming it `tag` is to make it agnostic, in case this same distinction is necessary in other cases (and because, essentially, we're "labeling" specific elementary flows, in this case based on their origin). Implementation-wise, the uuid generator could be refactored in a way that the path is the same when the tag is empty, and different if it has any value. It would however require of extra columns on the mappings to convert to and from FEDEFL
* We considered whether `context` could carry this information, but came to the conclusion that it's too different to the actual context for it to make sense.
* To explore all possibilities: it could also be encoded in the `unit`, thus having several e.g. Carbon dioxide, with different units (`kg CO2 eq`, `kg CO2 eq (biogenic)`, and `kg CO2 eq (land use)`. This is not the most intuitive place to look for the information, but would be similar in functionality to add it to the name.
* Ignore this, and force different stakeholders using FEDEFL to use their own implementation of the above. I believe this would make the "forced alternative versions" a weakness of FEDEFL.

@bl-young I would love to hear your view on this. At the moment, we're unfortunately exploring the last option, but it's our least preferred one, since we'd be diverging from FEDEFL, and it creates an incentive for us to use a different nomenclature instead (to guarantee interchangeability).

**Reply by bl-young** [reply]
> I'm assuming (please correct me if I'm wrong) that this is because the _flowable_ itself is not different, chemistry-wise, so conceptually it makes the flowable list messier.

Yes this is the historical justification I believe. I'm hoping @WesIngwersen will chime in with his thoughts.

**Reply by WesIngwersen** [reply]
Thank you for starting this discussion thread.  The need for biogenic carbon tracking in data using the FEDEFL has surfaced before.  Since then, this need has only increased.   So I agree that we need to address this.  This decision will hopefully be made by the Federal LCA Commons Agency representatives in consultation with external experts like yourselves.  I can't offer a firm timeline for this right now but we can update this thread. 

FYI @rferaldipnnl @PeterArbuckle-USDA @acarpent @sarahcashmanz

**Comment by bnjmnmorelli** [comment]
Proposed FEDEFL Update: Add flows to track biogenic carbon

The FEDEFL curation team has recently been considering the addition of flows to track biogenic carbon. We would appreciate your general feedback and suggestions on nomenclature. Thanks to Earthster for resurfacing this issue recently as documented [here](https://docs.earthster.org/en/articles/12808982-environmental-exchanges-in-earthster).

Justification:

- Enable biogenic C modeling with standard FEDEFL flows.
- Compliance with ISO 14067.
- Better compatibility with ecoinvent and ILCD flow lists.

Implementation:

- The table below outlines proposed flows for modeling fossil/non-fossil carbon. ‘Carbon, fossil’ is the primary emission flow included in inventories. ‘Carbon dioxide, to/from soil or biomass stock’ is used to model storage and the release of stored carbon. The FLCAC will continue to apply the net-zero assumption to biogenic/non-fossil carbon dioxide emissions (characterization factor = 0). Tracking biogenic carbon dioxide is optional.
- Using dedicated LCIA methods, the ‘Carbon dioxide’ resource/air flow can be used with ‘Carbon dioxide, non-fossil’ and ‘Carbon dioxide, fossil’ to explicitly track carbon flows.
- From a GHG perspective, carbon monoxide flows are relevant for LCIA methods a) through consideration of short-lived climate factors or b) by assuming that CO oxidizes to CO2. TRACI and ReCiPe exclude CO.
- Flows that will be replaced and deleted, “Carbon dioxide” and “Methane,” will be retained until LCIA methods and processes have been sufficiently reviewed and updated. While they remain in the FEDEFL, they will continue to be characterized assuming they are fossil.
- Existing LCI data and LCIA methods will be updated to reflect new GHG flows.

<img width="595" height="778" alt="image" src="https://github.com/user-attachments/assets/095ef17c-eb5c-40db-a983-9cd45e033148" />

**Reply by WesIngwersen** [reply]
Should we assume that the proposed names in the left column are the proposed Flowable name?  And that there are no proposed Context changes?

**Reply by bnjmnmorelli** [reply]
That is correct, but names are not yet finalized. The current proposal favors explicit labeling, 'fossil' and 'non-fossil', for emission flows but does not differentiate for the resource flow. Biogenic is a potential alternative to non-fossil.
